Octokit.Tests.Integration/Clients/OrganizationActionsClientTests.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Octokit.Tests.Integration.Clients
+{
+    public class OrganizationActionsClientTests
+    {
```

Comment:
Currently the OrganizationActionsClient exists to have a property for the OrganizationSecretsClient so that other operations could be added (as well as other child clients) at a later date without needing to refactor the code to enable it.  There is no actual integration type logic that is done in that client based on what I created.  I created this file so that it existed and could be used later if needed, but shows that it wasn't forgotten or overlooked.